### DISCUSSION TITLE
Separate Record/Play/Stop commands in slot transport action

### DISCUSSION THREAD

**Discussion by johnsamuelson** [helgoboss/helgobox#1177] [Ideas]
Hi,

I have a MIDI controller set to send a CC to Playtime, via Realearn, to record into a slot. Playtime is set to a pre-set length of 4 bars.

I want to be able to press the 'rec' CC button to put Playtime into record mode, then record 4 bars, then select a different slot and do the same and so on.

However I can only get it to record by sending a permanent 127 message; when I move to the second column, I have to reset the button press to 0 and then back to 127. This is fine if I've moved to the correct column, but if I haven't, I am likely to stop the playback of the first column instead.

It doesn't make much sense to me to have to map one button to all 3 commands. Is it possible to have a separate mapping for all 3, ie one CC for record, one CC for play, one CC for stop? I'm just guessing based on the previous state otherwise, which doesn't feel very reliable, or elegant. 

Thanks!

John

**Comment by helgoboss** [comment]
I think what you are asking for already exists:

- "Play/stop" will just play (as long as you react to "Press only")
- "Stop" will just stop
- "Record/stop" will just record (as long as you react to "Press only")

**Reply by johnsamuelson** [reply]
Ah great, thanks. I have to confess that my workings in ReaLearn are pretty rudimentary so tips like this are very useful
